internal/v1/api.yaml

```diff
@@ -581,6 +612,44 @@ components:
             Array of exactly one image request. Having more image requests in one compose is currently not supported.
         customizations:
             $ref: '#/components/schemas/Customizations'
+    CreateBlueprintRequest:
+      type: object
+      additionalProperties: false
+      required:
+        - name
+        - description
+        - distribution
+        - image_requests
+        - customizations
+      properties:
+        name:
+          type: string
+          example: "My Blueprint"
+          maxLength: 100
+        description:
+          type: string
+          example: "My blueprint description"
+          maxLength: 250
+        distribution:
+          $ref: '#/components/schemas/Distributions'
+        image_requests:
+          type: array
+          minItems: 1
+          maxItems: 1
+          items:
+            $ref: '#/components/schemas/ImageRequest'
+          uniqueItems: true
+          description: |
+            Array of exactly one image request. Having more image requests in a single blueprint is currently not supported.
```

Comment:
This is probably leftover from a copy/paste... shouldn't there be a request per target?

Reply:
This is copy-pasted from the "create compose" endpoint without changes, the way I understand this is that this only accepts array with exactly one element and this is for the future (e.g. when multiple requests can be send). @croissanne ?

Reply:
So we'd generate multiple compose requests from the blueprint, so this will have to change in future for sure (but we can change stuff under /experimental so).

And then we also might arrive at a point where a single compose requests can have multiple upload targets, which will be the case for edge (s3 + pulp).

Reply:
So if you ask me @lucasgarfield I'd say out of scope for this PR :-D

Reply:
also I think it's fine to keep this as an array, for koji we ended up accepting multiple image requests per compose reqs so who knows where we need to go with this.